test/fancylogger.py

```diff
@@ -51,6 +52,24 @@ def classless_function():
     logger.warn("from classless_function")
 
 
+class FansyLoggerLogToFileTest(EnhancedTestCase):
+    """
+    Tests for fancylogger, specific for logToFile
+    These dont' fit in the FansyLoggerTest class because they don't require the setUp and tearDown
+    """
+
+    def test_logtofile(self):
+        """Test to see if logtofile doesn't fail when logging to a non existing file /directory"""
+        tempdir = tempfile.gettempdir()
+        non_dir = os.path.join(tempdir, 'verytempdir')
```

Comment:
well, you make sure it doesn't below anyway? the two lines above are better covered by `mkdtemp`